src/Resources/views/Block/block_search_result.html.twig

```diff
@@ -52,11 +52,31 @@ file that was distributed with this source code.
             </div>
             {% if results_count > 0 %}
                 <div class="box-body no-padding">
-                    <ul class="nav nav-stacked sonata-search-result-list">
+                    <ul class="nav nav-stacked sonata-search-result-list products-list">
                         {% for result in current_page_results %}
                             {% set link = admin.getSearchResultLink(result) %}
                             {% if link %}
-                                <li><a href="{{ link }}">{{ admin.toString(result) }}</a></li>
+                                <li class="item no-padding">
+                                    <a class="main pull-left" href="{{ link }}">{{ admin.toString(result) }}</a>
+                                    <span class="matches pull-right">
+                                        {% for name, filter in filters %}
+                                            {% set match = name|split('.')|reduce((carry, v) => carry ? attribute(carry, v) : null, result) %}
+
+                                            {#
+                                                Some model managers are not respecting the "sonata_admin.global_search.case_sensitive" configuration option.
+                                                If you see no matching filters for a search result, it is probably the reason. In that case,
+                                                you SHOULD consider using `false` as value for this option.
+                                            #}
+                                            {% if match and ((filter.options.case_sensitive is not same as(false) and filter.value.value in match)
```

Comment:
These filters are implementing [`FilterInterface`](https://github.com/sonata-project/SonataAdminBundle/blob/f15b39ab3194d1b2dae1ebf98a6daa7f9c5d76d7/src/Filter/FilterInterface.php) if I'm not mistaken, but there is no `getValue` method in the interface.

And [`getValue` from `Filter`](https://github.com/sonata-project/SonataAdminBundle/blob/f15b39ab3194d1b2dae1ebf98a6daa7f9c5d76d7/src/Filter/Filter.php#L216) is deprecated.

Reply:
Nice catch!
I've replaced these usages with a new variable holding the search term.
Thank you.